app/uk/gov/hmrc/cgtpropertydisposalsfrontend/views/landing_page.scala.html

```diff
@@ -98,16 +98,6 @@ <h2 class="heading-medium" id="howTo">@messages("landingPage.howTo.h2")</h2>
         <li>@messages("landingPage.howTo.list1.bullet3")</li>
     </ul>
 
-    <div class="panel panel-border-wide">
```

Comment:
this needs to be feature flagged - if amend returns is enabled we want this and if it's disabled we want the old content. There already exists a flag you can access via ViewConfig called `amendReturnsEnabled` or something